packages/app/src/providers/h5grove/models.ts

```diff
@@ -1,7 +1,6 @@
 import { type AttributeValues, type Filter } from '@h5web/shared/hdf5-models';
 
 export type H5GroveEntityResponse = H5GroveEntity;
-export type H5GroveDataResponse = unknown;
```

Comment:
I've removed this type, as it was only used as return type to `fetchValue`. Now, if I write `toJSON(...) as H5GroveDataResponse` inside the `getValue` method, ESLint says the casting is unnecessary and removes it.